src/Features/Quotas/Sidenav/Sidenav.tsx

```diff
@@ -46,31 +46,14 @@ const SideInfo: React.FC<SideInfoProps> = ({ teams }) => {
           <p className={styles.info}>{`Showing ${teamsToDisplay.length} teams`}</p>
         </div>
       </div>
-      <section className={styles.taskSection}>
+      <SideNavLinks>
         {teamsToDisplay.map((team) => {
-          //@ts-ignore
-          return <Task team={team} isActive={globalMatch?.params?.teamId === team.id} />;
+          return (
```

Comment:
@LucasGrimauth nice use of the sidenav here, but it looks like we lost some of the padding. (specifically to the left)